docs/release-notes.rst

```diff
@@ -6,6 +6,71 @@
  Release Notes
 ###############
 
+**************
+ Version 0.36
+**************
+
+Version 0.36.0
+==============
+
+**Release Date:** August 22, 2024
+
+**New Features**
+
+-  WebUI: In the enterprise edition of Determined, when RBAC is enabled, allow only Viewer, Editor,
+   GenAI, and Workspace Admin roles to view resource quotas for each workspace in the WebUI. When
+   RBAC is not enabled, any user can view resource quotas.
+
+-  RBAC: Add a pre-canned role called ``EditorProjectRestricted`` that supersedes the ``Viewer``
+   role and precedes the ``Editor`` role.
+
+   -  Like the ``Editor`` role, the ``EditorProjectRestricted`` role grants the permissions to
+      create, edit, or delete experiments and NSC (Notebook, Shell or Command) type workloads within
+      its scope. However, the ``EditorProjectRestricted`` role lacks the permissions to create or
+      update projects.
+
+-  Kubernetes: Add experimental support for AMD ROCM GPUs. To use, set ``slotType=rocm``. Visit
+   :ref:`helm-config-reference` for more details.
+
+-  Images: Add New ROCM 6.1 images with DeepSpeed for MI300x users. Dev versions of these images can
+   be found in our Docker Hub, under `pytorch-infinityhub-dev
+   <https://hub.docker.com/repository/docker/determinedai/pytorch-infinityhub-dev/tags>`__ and
+   `pytorch-infinityhub-hpc-dev
+   <https://hub.docker.com/repository/docker/determinedai/pytorch-infinityhub-hpc-dev/tags>`__.
+   Users can build these images locally based on the Dockerfiles found in our `environments
+   repository
+   <https://github.com/determined-ai/environments/blob/main/Dockerfile-infinityhub-pytorch>`__.
+
+-  Master: Add a ``ui_customization`` option to the :ref:`master configuration
+   <master-config-reference>` for specifying a custom logo for the WebUI.
+
+**Bug Fixes**
+
+-  Experiments: Report an experiment's status as FAILED if any failure occurs during the shutdown
+   process, before the experiment has completed gracefully.
```

Comment:
Slight nit: Instead of gracefully, do we mean successfully? @corban-beaird @tara-det-ai defer to you on wording but I am confused what that means

Reply:
this bug fix note is for #9693 

This wording accurately reflects the distinction between a successful, orderly shutdown and a failure that prevents it. "successful + orderly" = "graceful".